portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,34 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*; 
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    ArrayList<String> list = new ArrayList<String>();
+    list.add("My name is Miah");
+    list.add("Im from NYC");
+    list.add("I love coding!");
+
+    // Send the JSON as the response
+    String json = convertToJsonUsingGson(list);
```

Comment:
as a small nit I'd say we probably don't need this helper method - as we can just in one line say:
String jsonComment = new Gson().toJson(list);
and that one liner is pretty self explanatory